README.md

```diff
@@ -105,14 +105,17 @@ The CLI argument `--config` is not required.
 If it is not provided, `markdownlint-cli` looks for the file `.markdownlint.json`/`.markdownlint.yaml`/`.markdownlint.yml` in current folder, or for the file `.markdownlintrc` in the current or all parent folders.
 The algorithm is described in detail on the [`rc` package page][rc-standards].
 If the `--config` argument is provided, the file must be valid JSON, JSONC, JS, or YAML.
-JS configuration files contain JavaScript code, must have the `.js` extension, and must export (via `module.exports = ...`) a configuration object of the form shown above.
+JS configuration files contain JavaScript code, must have the `.(c)js` extension, and must export (via `module.exports = ...`) a configuration object of the form shown above.
+So, if your workspace _(project)_ is [ESM-only] _(`"type": "module"`set in the root 'package.json' file)_, then the configuration file **should end with `.cjs` file extension**.
 A JS configuration file may internally `require` one or more npm packages as a way of reusing configuration across projects.
 
 `--enable` and `--disable` override configuration files; if a configuration file disables `MD002` and you pass `--enable MD002`, it will be enabled.
 If a rule is passed to both `--enable` and `--disable`, it will be disabled.
 
 > JS configuration files must be provided via the `--config` argument; they are not automatically loaded because running untrusted code is a security concern.
 
+[ESM-only]:
```

Comment:
Missing link?

Reply:
Fixed. I added a link to the popular gist, which probably has all the answers regarding ESM vs CJS workspace and usage in Node.js projects.